Gracekeza/challenge1
# #169456122 Re-design the UI according to LFA's feedback

#### What does this PR do?
Have re-designed UI according to LFA's feedback
#### Description of Task to be completed?
Have the following changes made:
`Add details on login and signup page`
`re-design the new story layout`
`Make the navigation bar across all pages`
`Show menu Icon on smaller screens`
`Centering stories on tablets`
#### How should this be manually tested?
By visiting [MyDiary](https://gracekeza.github.io/challenge1/UI/login.html),
Verify if all tasks have been completed
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#169456122
#### Screenshots (if appropriate)
## Before
![Screenshot (100)](https://user-images.githubusercontent.com/20791990/67887165-7a0c6200-fb53-11e9-9cdd-b973240d8ad8.png)
![Screenshot (101)](https://user-images.githubusercontent.com/20791990/67887169-7aa4f880-fb53-11e9-8984-5c8ddc22f2f7.png)
![Screenshot (102)](https://user-images.githubusercontent.com/20791990/67887170-7bd62580-fb53-11e9-8045-64eef13d002c.png)
![Screenshot (103)](https://user-images.githubusercontent.com/20791990/67887172-7c6ebc00-fb53-11e9-9eb2-096d9e575dda.png)
## After
![Screenshot (104)](https://user-images.githubusercontent.com/20791990/67887855-c0ae8c00-fb54-11e9-9ddc-5a574b1e428c.png)
![Screenshot (105)](https://user-images.githubusercontent.com/20791990/67887859-c1472280-fb54-11e9-9453-6db67a23f1c3.png)
![Screenshot (106)](https://user-images.githubusercontent.com/20791990/67887860-c2784f80-fb54-11e9-9612-20d91e6e2707.png)
![Screenshot (107)](https://user-images.githubusercontent.com/20791990/67887861-c2784f80-fb54-11e9-9e3c-68f7c993b803.png)
![Screenshot (108)](https://user-images.githubusercontent.com/20791990/67887865-c4daa980-fb54-11e9-9171-5d1cebada7da.png)
![Screenshot (109)](https://user-images.githubusercontent.com/20791990/67887866-c5734000-fb54-11e9-9d21-5fbe311427b4.png)

#### Questions:
